src/Pieces/Piece.h

```diff
@@ -1,31 +1,38 @@
 #pragma once
 
+#include "Board/Board.h"
 #include <stdbool.h>
+#include <stdint.h>
+
+#define MASK_PIECE(pos) (0x0000000F << pos)

```

Comment:
"best practice" is to prefix defines with the file name to try and avoid name collisions so would prefer `PIECE_MASK`

Reply:
🤓